inc/objectlock.class.php

```diff
@@ -311,20 +311,21 @@ private function lockObject() {
          // add a script to unlock the Object
          echo Html::scriptBlock( "$(document).ready( function() {
                   $(window).on('beforeunload', function() {
-                     //debugger ;
+                     //debugger;
                       $.ajax({
                           url: '".$CFG_GLPI['root_doc']."/ajax/unlockobject.php',
                           async: false,
                           cache: false,
                           data: 'unlock=1&id=$id'
                           });
```

Comment:
We have an indentation problem here.
The above line & the 2 next need to be unindented